content/blog/posts/2022-o3de-user-survey-report.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "2022 O3DE User Survey Report" 
+date: 2022-10-12
+slug: 2022-o3de-user-survey-report
+author: Yuyi Hsu
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+---
+#### On a Mission to Continuously Improving the User Experience in O3DE
+
+The O3DE SIG UI-UX conducted a survey in June 2022 with the goal to continuously improve the Open 3D Engine (O3DE) experience for developers, artists, content creators and other users. Results from over 200 participants provided a snapshot that will be used to guide the evolution of features, capabilities and enhancements that will be included in future releases to improve the user experience in O3DE. 
+
+The survey collected user feedback on the O3DE user experience based on nine common game development workflows, from Onboarding, Game Design, Actor Development, World Building, Look Development, Multiplayer, to Team Collaboration, Engine Extension, and Packaging & Deployment. Within each workflow, users were asked to rate:
+
+- Workflow satisfaction: how satisfied they are with the workflow
+- Workflow confidence: how confident they feel they achieved their stated objectives after completing the workflow
+- Workflow efficiency: how efficient they feel their time spent on the workflow
+- Feature satisfaction: how satisfied they are using the features within the workflow.
+
+#### What Did We Find?
+
+Results from the [final survey report](https://github.com/o3de/sig-ui-ux/blob/main/user-research/2022%20Q2%20O3DE%20User%20Experience%20Survey.md) suggest areas of improvement across all nine workflows surveyed. The two highest performing workflows are Onboarding and Look Development, which suggests the data-driven plan to focus on improvements for these two workflows has been received positively, enabling users to finish these workflows within a reasonable amount of time. Meanwhile, six workflows, including Actor Development, World Building, Multiplayer, Team Collaboration, Engine Extension, Packaging and Deployment, received lower ratings, which suggests these workflows require the greatest improvement.
+
+{{< image-width src="/images/blog/user-survey-2022/survey-2022.png" width="100%" alt="Mean Ratings Across 3 User Experience Metreics" >}}
+
+We also asked users to rate their experience with the features within the workflow. Based on the data, the Help System across O3DE, the integration with the version control tools, the toolings (e.g. Terrain and Dynamic Vegetation) in the World Building workflow, the Physics System, and the asset management experience (including Asset Browser) received lower satisfaction ratings (ranging from 2.5 to 2.8). The UX performance of these features could be the reason that drove lower ratings in their designated workflows.
```

Comment:
I was repeatedly misreading "Based on the data, the Help System, ...", as though "data" was also part of the list. So in my head I read it as "Based on all of these items". 
I think rephrasing to something like "The data indicates that <item 1>, <item 2>, etc." is more straightforward to comprehend.

```suggestion
We also asked users to rate their experience with the features within the workflow. The data indicates that the Help System across O3DE, the integration with version control tools, the tooling for the World Building workflow (e.g. Terrain and Dynamic Vegetation), the Physics System, and the asset management experience (including Asset Browser) received lower satisfaction ratings, ranging from 2.5 to 2.8. The UX performance of these features could be the reason that drove lower ratings in their designated workflows.
```